frontend/src/components/viewers/Tensorboard.test.tsx

```diff
@@ -102,9 +103,14 @@ describe('Tensorboard', () => {
     const config = { type: PlotType.TENSORBOARD, url: 'http://test/url' };
     const getAppMock = () => Promise.resolve({ podAddress: 'test/address', tfVersion: '1.14.0' });
     jest.spyOn(Apis, 'getTensorboardApp').mockImplementation(getAppMock);
+    jest.spyOn(Apis, 'isTensorboardPodReady').mockImplementation(() => Promise.resolve(true));
     tree = shallow(<TensorboardViewer configs={[config]} />);
 
     await TestUtils.flushPromises();
+    jest.runOnlyPendingTimers();
+    await TestUtils.flushPromises();
+    expect(setInterval).toHaveBeenCalledTimes(1);
+    expect(setInterval).toHaveBeenCalledWith(expect.any(Function), 5000);
     expect(tree).toMatchSnapshot();
```

Comment:
Can you use `toMatchInlineSnapshot` instead? From now on, prefer always changing to inline snapshot and keep snapshot size small.

Reply:
Just a question for the long term: it might be confusing that we keep both snapshot in snapshot file and in test with toMatchInlineSnapshot. E.g., first, which snapshot should be in snapshot file and which part should be in test file? Only later added part in test file. Moreover,  gradually, keep toMatchInlineSnapshot will boost the test file size and makes the test logic difficult to read.  Kind of having test logic and snapshot snippet in the same files look quite confusing to me. Maybe it would better we move away from snapshot file completely?

Reply:
Oh, one more thing, there is actually no snapshot diff here...

Reply:
SG if there is no snapshot change.

Let's have a chat about snapshot testing then.